pkg/gen/input/makefile/internal/file/Makefile.gen.app.mk.template

```diff
@@ -6,7 +6,7 @@ YQ=docker run --rm -u $$(id -u) -v $${PWD}:/workdir mikefarah/yq:4.29.2
 HELM_DOCS=docker run --rm -u $$(id -u) -v $${PWD}:/helm-docs jnorwood/helm-docs:v1.11.0
 
 ifdef APPLICATION
-DEPS := $(shell find $(APPLICATION)/charts -name "Chart.yaml" -printf "%h\n")
+DEPS := $(shell find $(APPLICATION)/charts -regex "$(APPLICATION)/charts/[-A-Za-z]*/Chart.yaml" -printf "%h\n")
```

Comment:
I'd recommend using `-maxdepth` to avoid hidden corner cases (like numbers in the chart name :) ). 

I've tested it with `-maxdepth 2` and works fine:

```
find helm/kyverno/charts -maxdepth 2 -name "Chart.yaml" -printf "%h\n" 
helm/kyverno/charts/kyverno
helm/kyverno/charts/policy-reporter
```

wdyt?

Reply:
Oh, yeah, that's even better